include/yaml-cpp/node/convert.h

```diff
@@ -241,6 +242,35 @@ struct convert<std::list<T> > {
   }
 };
 
+// std::array
+template <typename T, std::size_t S>
+struct convert<std::array<T, S> > {
+  static Node encode(const std::array<T, S>& rhs) {
+    Node node(NodeType::Sequence);
+    for (typename std::array<T, S>::const_iterator it = rhs.begin();
+        it != rhs.end(); ++it)
+      node.push_back(*it);
+    return node;
+  }
+
+  static bool decode(const Node& node, std::array<T, S>& rhs) {
+    if (!node.IsSequence())
+      return false;
+
+    std::size_t index = 0;
+    for (const_iterator it = node.begin(); it != node.end(); ++it) {
```

Comment:
I think it'd be more readable to iterate through the sequence with an index, rather than iterator.